### PR TITLE
re2 20230601

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/re2.rb [hunk 1]
```diff
@@ -29,10 +29,9 @@ class Re2 < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "abseil"
```
Comment (chenrui333) [inline Formula/re2.rb:32]: relates to https://github.com/google/re2/commit/552c2b8f737f871b8bd00006e0f5d4036c5a5683

**Comment by carlocab** [comment]
On the next run, we can probably remove this:

https://github.com/Homebrew/homebrew-core/blob/8fa799125c889c647d49ba9507117423c6e27022/Formula/re2.rb#L36-L37

`bloaty` will likely need a patch.

File: Formula/or-tools.rb [hunk 2]
```diff
@@ -101,3 +102,29 @@ def install
 +#endif
  
  #endif  // OR_TOOLS_BASE_LOGGING_H_
+diff --git a/ortools/lp_data/lp_parser.cc b/ortools/lp_data/lp_parser.cc
```
Comment (carlocab) [inline Formula/or-tools.rb:105]: Upstreamed at https://github.com/google/or-tools/pull/3840

**Comment by carlocab** [comment]
Should be good now.

**Review by carlocab — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**